node/node.d.ts

```diff
@@ -496,6 +496,8 @@ declare module "http" {
         path?: string;
         headers?: { [key: string]: any };
         auth?: string;
+        requestCert?: boolean;
+        rejectUnauthorized?: boolean;
```

Comment:
is this right? this is http package. not https package.
https://github.com/tinganho/DefinitelyTyped/blob/cce63453c02d4c03e757fd84116e1838645582c3/node/node.d.ts#L823